src/git_auto_export/BUILD

```diff
@@ -0,0 +1,18 @@
+python_sources(name="edx-git-auto-export")
+
+python_distribution(
+    name="git_auto_export",
+    dependencies=[":edx-git-auto-export"],
+    provides=setup_py(
+        name="edx-git-auto-export",
```

Comment:
I have tried to follow the same name convention but If I change the package name to something else it says `ModuleNotFound git_auto_export` so in order to fix it, I used that name. I will share it screenshot soon

Reply:
I believe that might have something related to how the setup file is generated or the usage within the package itself. 

AFAIK, The name shouldn't really break things, the problem would be that it might not have been reflected on other placed properly.